projects/knitfinance/index.js

```diff
@@ -1,44 +1,129 @@
 const axios = require("axios");
 
-let url = "https://adminv1.knit.finance/api/tvl";
+let url = "https://adminv1.knit.finance/api/public/tvl";
```

Comment:
Code can be shortened a bit:

    const { get } = require("../helper/http")
    const { toUSDTBalances } = require('../helper/balances')

    const url = "https://adminv1.knit.finance/api/public/tvl/"

    const chainConfig = {
      bsc: 'bsc',
      polygon: 'matic',
      ethereum: 'eth',
      heco: 'heco',
      fantom: 'fantom',
      avalanche: 'avalanche',
      kcc: 'kcc',
      harmony: 'harmony',
      okexchain: 'okexchain',
      syscoin: 'syscoin',
      telos: 'telos',
      moonriver: 'moonriver',
      milkomeda: 'milkomeda',
      moonbeam:'moonbeam',
      bitgert: 'bitgert',
      xdai: 'gnosis',
      reef: 'reef',
    }

    module.exports = {
      timetravel: false,
    }

    function addChain(chain){
      module.exports[chain] = {
        tvl: async () => {
          const key = chainConfig[chain]
          let response = (await get(url + key))
          return toUSDTBalances(response.data.data.tvl[key])
        }
      }
    }

    Object.keys(chainConfig).map(addChain)